### ISSUE TITLE
examples/exporter: PrintExporter output is not clear 

### ISSUE THREAD

**Issue by rogpeppe** [census-instrumentation/opencensus-go#762]
When I run the helloworld_server example, it prints lines looking like this:

    2018/05/24 14:55:49 &{0xa14a80 2018-05-24 14:55:28.201596042 +0100 BST m=+1.004805399 2018-05-24 14:55:49.202266421 +0100 BST m=+22.005475512 []}

This is because PrintExporter just uses log.Println(SpanData), which doesn't look great.
It might be nice if the example used an output format that made it clearer what information is available to the exporter.

**Comment by ramonza** [comment]
Important to remember that this is just for illustrative examples. Maybe we should have a true log exporter that's designed for production use with features like templating of the log message?

**Comment by vaijab** [comment]
> Maybe we should have a true log exporter that's designed for production

This could be an interesting feature to spec out. I am actually looking into some logger that logs at request (gRPC) level. It would be pretty cool to have a logger that could actually log selected fields from a trace, i.e. traceId, spanId, attributes, etc.

I would be happy to contribute to a spec and/or code. :-)

**Comment by ramonza** [comment]
Repurposing this for a production-quality structured logging exporter.

**Comment by savaki** [comment]
I'm interested in picking this one up.  We ended already doing something like this internally.  Here's an example of what it looks like (census is our wrapper for opencensus)

```go
census.Info(ctx, "message",
   census.String("key", "value"),
   census.Int("id", "123"),
);
```

**Comment by ramonza** [comment]
Retitled to make it clear that this issue is around supporting the existing {view,trace}.Exporter interfaces rather than creating a new signal type.

**Comment by ramonza** [comment]
Yup, happy to restore the original purpose. If we want to do a zap exporter, it would probably get it's own repo.

**Comment by rghetia** [comment]
fixed by #911
